app/policies/user_policy.rb

```diff
@@ -16,4 +16,8 @@ def update_profile?
     user?
   end
 
+  def all?
+    sign_up?
```

Comment:
I do not like this type of polices checking. I think you should use understandable private method here. Like, user_is_admin? 
This chain in future can be messy. For example you decided to add another condition. Now it's 
`sign_up? || user.is_bookkeeper?`
Than you have another query, like active users - users that have completed tasks on some period, for some kind of report - `active_this_month`
So now it's 
`def active_this_month?
  all?
end`

But what if at some point of time you decide to make sign_up available for everyone. You look at `active_this_month` and it seems OK but it's not secure now.

I think you should refactor all polices with rule 'do not call another polices inside policy check'.

Reply:
yeah, it's a question about similar operations. I didn't have problem with using my approach, but it's hard to follow if you are new on the project.

I'll change this.